lib/ex_doc.ex

```diff
@@ -7,23 +7,24 @@ defmodule ExDoc do
     @moduledoc """
     Configuration structure that holds all the available options for ExDoc
 
-    You can find more details about this options in the `ExDoc.CLI` module.
+    You can find more details about these options in the `ExDoc.CLI` module.
     """
+
     defstruct [
       assets: nil,
       canonical: nil,
       deps: [],
       extra_section: nil,
       extras: [],
       filter_prefix: nil,
-      formatter: "html",
+      formatter: Mix.Project.config[:default][:formatter],
```

Comment:
sorry, I was too tired yesterday and overlooked this.
Please have a look again at the rebased commit